bungiesearch/management/commands/search_index.py

```diff
@@ -132,9 +132,11 @@ def handle(self, *args, **options):
                 mapping = {}
                 for mdl_idx in src.get_model_indices(index):
                     mapping[mdl_idx.get_model().__name__] = mdl_idx.get_mapping()
-
+                
+                analysis = mdl_idx.collect_analysis()
```

Comment:
I may not cover all cases here, though this fits my use case. If I try to do the same thing with analysis as was done with mapping, I get a parser exception. Let me know if you have ideas --- I'll reexamine in the morning.